gnoland/website/pkgs/doc/doc.go

```diff
@@ -0,0 +1,82 @@
+package doc
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"sort"
+	"strings"
+)
+
+func New(pkgPath string, files map[string]string) (*Package, error) {
+	p := Package{
+		ImportPath: pkgPath,
+		Filenames:  make([]string, 0, len(files)),
+	}
+
+	fset := token.NewFileSet()
+	gnoFiles := make(map[string]*ast.File)
+
+	for filename, fileContent := range files {
+		p.Filenames = append(p.Filenames, filename)
+
+		f, err := parser.ParseFile(fset, filename, fileContent, parser.ParseComments)
+		if err != nil {
+			return nil, err
+		}
+
+		ast.FileExports(f)
+
+		if strings.HasSuffix(filename, "_test.gno") {
+			continue
+		}
+
+		gnoFiles[filename] = f
+
+		if f.Doc != nil {
+			doc := f.Doc.Text()
+			if p.Doc != "" {
+				p.Doc += "\n"
+			}
+			p.Doc += doc
```

Comment:
Many package have top-level comments like

```
// PKGPATH: gno.land/r/test
```

Resulting in docs like this:

![image](https://user-images.githubusercontent.com/4681308/220447110-b1276c18-c708-460b-8428-6d5cd328a9da.png)

This should be fixed by ignoring `_filetest.gno` files, but check this after you do that :)

Reply:
Resolved by [bd57775](https://github.com/gnolang/gno/pull/526/commits/bd577756e9906d5e8ec71beb4464688ff17cd7bd)